### ISSUE TITLE
How to use alov to train

### ISSUE THREAD

**Issue by Breakmydream** [tangyuhao/GOTURN-Tensorflow#10]
it no every imgae ground-truth bounding box

**Comment by sajalkaushik17** [comment]
pls tell how tointegrate the dataset with the model

**Comment by anjali1601** [comment]
Please tell how to make train_file.txt for ALOV dataset and IMAGENET dataset where to get the bounding box numbers? or is it already available as there will be number of entries in this text file

**Comment by tangyuhao** [comment]
I did not write code to directly train the model using the dataset. What I did is to use the ground truth bounding box information in dataset lake ALOV and generate a new dataset which contains the input images my model needs and the ground truth labels. If you want to integrate the training process with the original dataset, you can write your own code to read the ALOV dataset and create trimmed input images on the fly.
